examples/splunk-sink/main.tf

```diff
@@ -19,11 +19,14 @@ provider "google" {
 }
 
 module "log_export" {
-  source               = "terraform-google-modules/log-export/google"
-  destination_uri      = module.destination.destination_uri
-  log_sink_name        = "test-splunk-sink"
-  parent_resource_id   = var.parent_resource_id
-  parent_resource_type = "project"
+  # source          = "terraform-google-modules/log-export/google"
```

Comment:
Please remove it

Reply:
@ingwarr, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-log-export/compare/895f1e300a9a019f610ae29eba23f0332dfc5955..7c44b7cceadce4ea5e3bc0cb4ef32d25c3f717e0)